### PR TITLE
cargo-fix: add page

### PR DESCRIPTION
**Pull request by Cioraz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by Cioraz** [comment]
Hey maintainers!
Just a small question, will this get merged into the code or is there an issue I must check into?

**Review by blueskyson — APPROVED** [review]

**Comment by blueskyson** [comment]
@Cioraz Thanks for contribution. This will be merged after 2 maintainers approve. Normally it takes 1 day to 3 months regarding the activity of the tldr community and the workload of the PR.

File: pages/common/cargo-fix.md
Comment (acuteenvy) [inline pages/common/cargo-fix.md:3]:
```suggestion
> Automatically fix lint warnings reported by `rustc`.
```

File: pages/common/cargo-fix.md
Comment (acuteenvy) [inline pages/common/cargo-fix.md:20]:
```suggestion
`cargo fix --test {{name}}`
```

File: pages/common/cargo-fix.md
Comment (acuteenvy) [inline pages/common/cargo-fix.md:36]:
```suggestion
- Run `n` jobs in parallel (default: number of logical CPUs):

`cargo fix --jobs {{n}}`
```

File: pages/common/cargo-fix.md
Comment (acuteenvy) [inline pages/common/cargo-fix.md:28]:
```suggestion
- Set the directory for all generated artifacts and intermediate files:

`cargo fix --target-dir {{path/to/directory}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
